client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
I'm confused.

The point of the hash is so that when I look at a single line of data I can uniquely identify across the entire class what assessment this line of data is from. Removing the case ID would require me to look at other lines of data to figure out which one is which based on the timestamp order, wouldn't it? This then is also potentially unreliable if randomization is eventually implemented.

Also this thread is getting long enough that I'm not sure where we are at in terms of what the data will look like. I've created a google doc documenting what I understand so far and if one of your could fill out the TODOs to make sure we are all on the same page that would be great.

https://docs.google.com/document/d/1vciz3dW373IatFtfGXv497EdR2-9njoZc_D-E8-9yzw/edit#